docs/gadgets/trace/tcpconnect.md

```diff
@@ -156,6 +156,117 @@ The gadget will show the connection and related information to it.
 
 ```bash
 $ sudo ig trace tcpconnect -c test-tcp-connect
-CONTAINER        PID     COMM             IP  SADDR            DADDR            DPORT
-test-tcp-connect 503650  wget             4   172.17.0.3       93.184.216.34    80
+CONTAINER        PID     COMM             IP  SADDR            DADDR            DPORT    SPORT
+test-tcp-connect 503650  wget             4   172.17.0.3       93.184.216.34    80       40658
+```
+
+### Calculating the latency of a connection
```

Comment:
Add a note somewhere documenting that using `--latency` will delay the connection event until the server replies (or timeout).